app/assets/stylesheets/custom.scss

```diff
@@ -73,9 +73,49 @@ a {
   vertical-align: right;
 }
 
+.text-listing-index {
+  display: inline-block;
+  width: 90%;
```

Comment:
Properties should be ordered display, margin-left, margin-right, margin-top, overflow, text-align, white-space, width